src/main/java/com/jfrog/ide/common/go/GoScanWorkspaceCreator.java

```diff
@@ -68,8 +69,17 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
             Path targetGoMod = targetDir.resolve(sourceDir.relativize(file));
             Files.copy(file, targetGoMod);
             goDriver.runCmd("run . -goModPath=" + targetGoMod.toAbsolutePath() + " -wd=" + sourceDir.toAbsolutePath(), true);
+            return FileVisitResult.CONTINUE;
+        }
+        // Skip .git, .idea and .vscode (deny list) directories.
+        if (StringUtils.containsAny(fileName, ".git", ".idea", ".vscode")) {
```

Comment:
Let's put the list of excluded directories in private static final class variables.